tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,803 @@
+an area of improvement - reducing traffic in the city and overall carbon 
+footprint associated with airport trips.
+
+Prior to instituting any programs, they would like you to more closely 
+examine trips to JFK (code 2) and Newark (code 3). For each airport, they 
+would like to know the following for the month of January:
+- Number of trips to that airport
+- Average trip duration (i.e drop off time - pickup time)
+- Average trip cost
+- Average tip
+- Minimum, Maximum and Average trip distance
+- Average number of passengers
+
+To do this, we can run the following query:
+
+```sql
+-- For each airport: num trips, avg trip duration, avg cost, avg tip, avg distance, min distance, max distance, avg number of passengers
+SELECT rates.description, COUNT(vendor_id) as num_trips,
+   AVG(dropoff_datetime - pickup_datetime) as avg_trip_duration, AVG(total_amount) as avg_total,
+   AVG(tip_amount) as avg_tip, MIN(trip_distance) as min_distance, AVG (trip_distance) as avg_distance, MAX(trip_distance) as max_distance,
+   AVG(passenger_count) as avg_passengers
+ FROM rides
+ JOIN rates on rides.rate_code = rates.rate_code
+ WHERE rides.rate_code in (2,3) AND pickup_datetime < '2016-02-01'
+ GROUP BY rates.description ORDER BY rates.description;
+```
+
+Which produces the following output:
+
+```sql
+-[ RECORD 1 ]-----+--------------------
+description       | JFK
+num_trips         | 225019
+avg_trip_duration | 00:45:46.822517
+avg_total         | 64.3278115181384683
+avg_tip           | 7.3334228220728027
+min_distance      | 0.00
+avg_distance      | 17.2602816651038357
+max_distance      | 221.00
+avg_passengers    | 1.7333869584346211
+-[ RECORD 2 ]-----+--------------------
+description       | Newark
+num_trips         | 16822
+avg_trip_duration | 00:35:16.157472
+avg_total         | 86.4633688027582927
+avg_tip           | 9.5461657353465700
+min_distance      | 0.00
+avg_distance      | 16.2706122934252764
+max_distance      | 177.23
+avg_passengers    | 1.7435501129473309
+```
+
+Based on your analysis, you are able to identify:
+- There are 13x more rides to JFK than Newark. This often leads to heavy traffic on the roads to and from JFK, especially during peak times. They’ve decided to explore road improvements to those areas, as well as increasing public transport to and from the airport (e.g, busses, subway, trains etc)
+- Each airport ride has on average the same number of passengers per trip (~1.7 passengers per trip).
+- The trip distances are roughly the same 16-17 miles.
+- JFK is about 30% cheaper, most likely because of NJ tunnel and highway tolls.
+- Newark trips are 22% (10 min) shorter.
+
+This data is useful not just for city planners, but also for airport travellers 
+and tourism organizations like the NYC Tourism Bureau. For example, a tourism 
+organization could recommend cost-conscious travelers who’d rather not fork out 
+$84 for a ride to Newark to use public transport instead, like the NJ Transit 
```

Comment:
> organization could recommend **to** cost-conscious travelers who’d rather not fork out 
$84 for a ride to Newark: **Use** public transport instead, ...

Reply:
Disagree with this grammatical suggestion.